src/rpc/rpccategory.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2018 The Bitcoin Core developers
```

Comment:
`// Copyright (c) 2018 The Bitcoin Core developers`

Reply:
done in https://github.com/bitcoin/bitcoin/pull/13945/commits/c9f15c12b24fbe39ea89a5b9d53f958923fc2ef4